### PR TITLE
Update weapons.json

### PR DESCRIPTION
**Pull request by randomdima**

### CODE REVIEW

File: configs/TI/weapons.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //try to keep names to just 1 word. Second word should be reserved for subtype - see "destroyerAfb"
 {
 	carrier:{
-		hit: 9
+		hit: 9 // making direct edit
```
Comment (NethIafin) [inline configs/TI/weapons.json:4]: don't use comments in JSON after first bracket

**Review by NethIafin — CHANGES_REQUESTED** [review]
This needs to be changed

File: configs/TI/weapons.json (hunk 1 above)
Comment (ServAlex) [reply configs/TI/weapons.json:4]: I don't see any

Comment (NethIafin) [reply configs/TI/weapons.json:4]: // making direct edit

**Pull request closed without merging**